### PR TITLE
9.1 paper edits

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

File: 9.1/paper.tex [hunk 1]
```diff
@@ -363,16 +361,16 @@ \subsection{Dedicated support for symbolic algebra, including algebra differenti
 and \texttt{SymmetricTensor} classes and associated functions. We have
 also implemented a set of utility functions with the following features:
 \begin{itemize}
-\item \texttt{differentiate} scalar expressions, tensor and symmetric tensors of expressions
-with respect to other scalar expressions, tensor and symmetric tensors of expressions;
```
Comment (jppelteret) [inline 9.1/paper.tex]: This change should not have been made :-(

Comment (kronbichler) [reply 9.1/paper.tex]: At least it wasn't obvious to me what the original text was supposed to say, in particular "tensor and symmetric tensorS of expressions"? Can you please explain?

Comment (jppelteret) [reply 9.1/paper.tex]: Each component of a `tensor of expressions` its itself a symbolic expression. In Mathematica, for example, one can write something like `C = A+B` where, `A`,`B` and `C` each represent a tensor. This is not what we support, but the new wording sounds like we do. I am busy working on new wording for the changed lines.

Comment (jppelteret) [reply 9.1/paper.tex]: See #30

**Pull request merged**